photon-lib/py/photonlibpy/photonPipelineResult.py

```diff
@@ -39,5 +39,10 @@ def getTimestamp(self) -> float:
     def getTargets(self) -> list[PhotonTrackedTarget]:
         return self.targets
 
+    def getBestTarget(self) -> PhotonTrackedTarget:
```

Comment:
I might just add a doc string explaining how "best" is calculated, and then we should be good to ship it